src/services/message.service.ts

```diff
@@ -70,6 +81,19 @@ export class MessageService {
       try {
         // Runs the sent function, with the data pulled from the emoji key.
         lambda(args.args);
+
+        embedData.emojiData = embedData.emojiData.filter((e) => e.emoji != reaction.emoji.name);
+
+        if (embedData.emojiData.length > minEmotes) {
```

Comment:
What exactly do minEmotes do?

Reply:
Once minEmotes are pressed on the message it will show the cutOff message.  No idea what a good name would be though.

Reply:
oh i see, its because you are changing the values of the params above, then checking.
I see why you are doing that now, typically i dont like changing the params of a function but here i think its suitable